app/views/categories/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= stylesheet_link_tag 'category_index' %>
+<% content_for :page_name, "Category" %>
+
+<%= render 'helper/navbar' %>
+<div class="container-fluid">
+  <% @categories.each do |category| %>
+    <div class="card mt-3 px-3">
+      <div class="card-body d-flex">
+        <div class="d-flex align-items-center" style="width: 80%;">
+          <img src="<%= category.icon %>" alt="category-icon" class="me-3 mr-3" style="max-width: 50px;" />
+          <h2><%= link_to category.name, category_path(category.id), class: "text-decoration-none second-color" %></h2>
+        </div>
+        <h3 class="second-color align-self-center" style="width: 20%;">$ <%= category.total %></h3>
+      </div>
+    </div>
+  <% end %>
+
+
+  <div class="button-container mt-5 d-flex align-items-center justify-content-center">
+    <%= link_to 'Add New Category', new_category_path, class: 'p-3 btn-background' %>
+  </div>
```

Comment:
- You have done well so far, but according to the project requirements, you are expected to stick to the design template for layout, colors, spacing etc. Please fix the position of this button and ensure it is placed at the bottom of the page as it is in the design template. Please fix this 🙏. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏 

Your design:
![wrgawrgawrg](https://github.com/AliRezaBmeDu/budget-app/assets/73607512/50ba4777-b9be-41d5-906e-41af7de2cb0f)

Design template:
![wq4rq2w4t](https://github.com/AliRezaBmeDu/budget-app/assets/73607512/1c7d26bb-848e-435a-94de-f09b9462d9cb)

Reply:
done